### ISSUE TITLE
Hands really like a mess

### ISSUE THREAD

**Issue by lucasjinreal** [magic-research/magic-animate#17]
mask doesn't provides enought information...

**Comment by zcxu-eric** [comment]
Hi, thanks for your interest on our work. That's true, we agree that mask cannot yield a clear hand and we are now still working on this part to improve face and hand regions. Will update soon once ready, stay tuned!

**Comment by lucasjinreal** [comment]
Consider add keypoints? It can given more accurate informations

**Comment by zcxu-eric** [comment]
> Consider add keypoints? It can given more accurate informations

Yes, we are now working on this. Please stay tuned, will release new checkpoints once ready.
